### PR TITLE
refactor: Apply code quality level 24 for Rector

### PR DESCRIPTION
**Pull request by samsonasik**
**Description**

Applied `SimplifyRegexPatternRector` to simplify regex pattern to known ranges

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Comment by samsonasik** [comment]
Ready to merge 👍

**Review by michalsn — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @michalsn for the review, let's merge ;)

File: tests/system/Database/Live/OCI8/GetFieldDataTestCase.php [hunk 1]
```diff
@@ -58,7 +58,7 @@ public function testGetFieldDataDefault(): void
         $fields = $this->db->getFieldData($this->table);
 
         $idDefault = $this->getFieldMetaData('id', $this->table)->default;
-        $this->assertMatchesRegularExpression('/"ORACLE"."ISEQ\$\$_[0-9]+".nextval/', $idDefault);
+        $this->assertMatchesRegularExpression('/"ORACLE"."ISEQ\$\$_\d+".nextval/', $idDefault);
```
Comment (kenjis) [inline tests/system/Database/Live/OCI8/GetFieldDataTestCase.php:61]: `\d` means a digit, but `[0-9]` means `0` to `9` (ASCII Arabic number). They are not the same.
Without `u` option, the result may be the exactly same, but the code implies different things.
So I don' t like this refactoring.

Comment (samsonasik) [reply tests/system/Database/Live/OCI8/GetFieldDataTestCase.php:61]: It doesn't have `/u` so should be safe, also, this is test so not apply to end user

**Pull request merged**